src/api/actions.ts

```diff
@@ -0,0 +1,12 @@
+import { apiClient } from './api';
+import { ITranslate } from '../components/Translate';
+
+export const getWord = async (): Promise<ITranslate> => {
+  try {
+    const response = await apiClient.get<ITranslate>('/translations');
+    const word = response.data;
+    return word;
+  } catch (err) {
+    throw new Error(err.message);
```

Comment:
Since right now in the api, we don't have proper error handling, try to throw an exception with a custom message inside a client and not relying on the server at the moment. 

Something like `throw new Error('Failed to fetch translations from the server')`.